servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -518,7 +534,7 @@ public boolean equals(Object object) {
   }
 
   private TraversalResult traverseToInode(String[] pathComponents, boolean persist)
```

Comment:
I think we can get rid of the persist input argument and always collect all the non-persisted inodes during the traversal